api/proto/teleport/legacy/types/webauthn/webauthn.proto

```diff
@@ -56,6 +56,36 @@ message SessionData {
   // "required".
   // An empty value is treated equivalently to "discouraged".
   string user_verification = 5 [(gogoproto.jsontag) = "userVerification,omitempty"];
+  // Scope authorized by this webauthn session.
+  ChallengeScope scope = 6 [(gogoproto.jsontag) = "scope,omitempty"];
+  // AllowReuse indicates that this session can be used multiple times for
+  // authentication, until the session expires.
+  bool allow_reuse = 7 [(gogoproto.jsontag) = "allow_reuse,omitempty"];
```

Comment:
This looks like a good candidate for an enum to me.

* https://protobuf.dev/programming-guides/api/#never-use-booleans-for-two-states
* https://cloud.google.com/apis/design/design_patterns#bool_vs_enum_vs_string